### PR TITLE
[bugfix] restore newlines at the end of setenv/prepend-path/set-alias statements

### PR DESCRIPTION
**Pull request by boegel**
The newlines at the end of various statements generated by the module generator were removed in #1060, but shouldn't have been, since easyblocks using the methods that generate these statements assume that the newlines are there.

@wpoely86: please review
cc @ehiggs

### CODE REVIEW

File: easybuild/tools/module_generator.py [hunk 1]
```diff
@@ -204,23 +204,23 @@ def load_module(self, mod_name):
             # not wrapping the 'module load' with an is-loaded guard ensures recursive unloading;
             # when "module unload" is called on the module in which the depedency "module load" is present,
             # it will get translated to "module unload"
-            load_statement = ["module load %(mod_name)s"]
+            load_statement = [self.LOAD_TEMPLATE, '']
         else:
             load_statement = [self.conditional_statement("is-loaded %(mod_name)s", self.LOAD_TEMPLATE, negative=True)]
-        return '\n'.join([''] + load_statement + ['']) % {'mod_name': mod_name}
```
Comment (wpoely86) [inline easybuild/tools/module_generator.py]: why are we dropping the empty line here?

Comment (boegel) [inline easybuild/tools/module_generator.py]: We're not. The string returned by `self.conditional_statement` already contains a newline, and for a non-conditional load the `''` will make sure it's followed by a newline

**Comment by wpoely86** [comment]
I think it's okay but it's quite hard to follow with all the newlines coming and going...

**Comment by boegel** [comment]
@wpoely86: it took me a while too, but the tests are checking things quite strictly (especially with the added test)

**Comment by boegel** [comment]
Hooray for Jenkins!

Thanks for reviewing @wpoely86!
